api/views.py

```diff
@@ -31,3 +33,18 @@ def get_object(self):
         queryset = Item.objects.filter(assigned_to=self.request.user)
         obj = get_object_or_404(queryset, serial_number=self.kwargs['pk'])
         return obj
+
+
+class SecurityUserViewSet(ModelViewSet):
+    serializer_class = SecuritySerializer
+    http_method_names = ['get']
+
+    def list(self, request, *args, **kwargs):
+        security_emails = SecurityUser.objects.all()
+
```

Comment:
@HawiCaesar consider using a list comprehension here . It's much faster and helps avoid making two lists and doing the append. This could be refactored as:
```
list_of_emails = [security_user.email for security_user in SecurityUser.objects.all()]
```